emscripten/ARToolKitJS.cpp

```diff
@@ -99,6 +99,14 @@ extern "C" {
 		NFT API bindings
 	*/
 
+	void matrixLerp(ARdouble src[3][4], ARdouble dst[3][4], float interpolationFactor) {
```

Comment:
We could try to use another lerp function, this maybe is not so efficient.
This function require **ARdouble**, not **float** data because **arFilterTransMat** function output **ARdouble**.
It also exist a lerp function from C++, we should see if we can use here in such a way.

Reply:
i am refering to this lerp https://en.cppreference.com/w/cpp/numeric/lerp

Reply:
they say that interpolation factor should be a value between 0.0 and 1.0

Reply:
see also this article on interpolation https://stackoverflow.com/questions/4353525/floating-point-linear-interpolation

Reply:
it seems that is some incompatibility with std::lerp and emscripten, i can not use it. i will abandon this idea.